### PR TITLE
File Added : All changed files

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

**Review by Ashwin2812 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: urmila/HTML/Assignment1/Q4.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <title>Assighment 4</title>
+  </head>
+<body>
+<P>
+	I am Urmila Gagare, I am 24 years old.
+	I am from Mumbai.
+    my Qualification BCA in computer.
+</P>
+  </body>
+  </html>
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment1/Q4.html]: clear indentation

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make proper indentation

File: urmila/HTML/Assignment1/Q1.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Assignment 1 </title>
+</head>
+  <body>
+  <h1>HTML</h1>
+  <h3>introduction</h3>
+  <p> HTML is a standard markup language for web page </p>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q1.html:14]: indentation missing

File: urmila/HTML/Assignment1/Q3.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  	Notepad or TextEdit
+  </h5>
+  <p>
+  	web pages can be created and modified by using profssional HTML editors.
+  	<br>
+  	However,for learing HTML we recommend a simple text Notepad(PC) or TextEdit(MAC)
+  </p>
+  </body>
+  </html>
+  <head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q3.html]: indentation missing

File: urmila/HTML/Assignment1/Q4.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  <title>Assighment 4</title>
+</head>
+<body>
+  <P>
+    I am Urmila Gagare, I am 24 years old.
+    I am from Mumbai.
+    my Qualification BCA in computer.
+  </P>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q4.html:8]: indentation missing

File: urmila/HTML/Assignment2/Q1.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  <p>
+    <small>
+      Google's initial public offering (IPO) took place five year later, on 19<sup>th</sup> August 2004.
+      <br>At that time Larry Page, Sergey Brin, and Eric Schmid adreed to work together at Google for <del>30</del>
+      <u>20</u> years, until the year 2004.
+    </small>
+  </p>
+  <h2 style="text-allign: center; color:  green;">Thank You!! </h2>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment2/Q1.html:32]: indentation missing

File: urmila/HTML/Assignment3/Q1.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+                <ul>
+                    <li>
+                        Triveni
+                    </li>
+                    <li>Kalyani</li>
+                </ul>
+            </dd>
+        </dl>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q1.html:67]: indentation missing

File: urmila/HTML/Assignment3/Q2Contact.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us .Contact us.Contact
+  </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2Contact.html:38]: html tag is missing 
indentation missing

File: urmila/HTML/Assignment3/Q2Home.html [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.
+    </p>
+  </body>
+  </html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2Home.html]: html tag is missing
indentation missing

File: urmila/HTML/Assignment3/Q2Mean.html [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    <li style="color: black">Hruday</li>
+    <li style="color: black">Aniket</li>
+  </ul>
+  <ol>
+    <li style="color: black">Bipin</li>
+    <li style="color: black">Priyanka</li>
+    <li style="color: black">Yashasvi</li>
+  </ol>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2Mean.html:39]: indentation missing

File: urmila/HTML/Assignment4/Q1.html [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+        </tr>
+        <tr>
+            <th>Female</th>
+            <td>1.9</td>
+            <td>0.003</td>
+            <td>43%</td>
+        </tr>
+    </table>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q1.html:66]: indentation missing

File: urmila/HTML/Assignment4/Q2.html [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+        </tr>
+        <tr class="table2">
+            <th>4:30</th>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+</body>
+</table>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q2.html:97]: indentation missing
body tag needs to be in end of file

File: urmila/HTML/Assignment4/Q2.html [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+            <td class="table3"></td>
+        </tr>
+        <tr class="table2">
+            <th>4:30</th>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+</body>
+</table>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q2.html]: move one line up this tag

File: urmila/HTML/Assignment4/Q3.html [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+    </tr>
+    <tr>
+      <td>Wednesday</td>
+      <td class="table3">8:00 a.m.</td>
+      <td class="table4">12:00 p.m.</td>
+      <td>XSL Formatting Object</td>
+    </tr>
+  </table>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q3.html:86]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: urmila/HTML/Assignment3/Q2Contact.html [hunk 14]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+    <head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2Contact.html:3]: opening HTML tag is missing

File: urmila/HTML/Assignment3/Q2Home.html [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+          subheading.This is home page subheading.This is home page subheading.This
+          is home page subheading.This is home page subheading.This is home page
+          subheading.This is home page subheading.This is home page subheading.This
+          is home page subheading.This is home page subheading.This is home page
+          subheading.This is home page subheading.This is home page subheading.This
+          is home page subheading.This is home page subheading.This is home page
+          subheading.This is home page subheading.
+      </p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2Home.html:48]: indentation missing

File: urmila/HTML/Assignment4/Q2.html (hunk 12 above)
Comment (Pawan1005) [reply urmila/HTML/Assignment4/Q2.html]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**